packages/langchain/src/openai/chat.test.ts

```diff
@@ -332,6 +338,160 @@ describe('Chat client', () => {
       expect(finalOutput).toMatchSnapshot();
     });
 
+    it('supports auto-streaming responses', async () => {
+      mockInference(
+        {
+          data: {
+            messages: [
+              {
+                role: 'user' as const,
+                content: 'What is the capital of France?'
+              }
+            ],
+            stream: true,
+            stream_options: {
+              include_usage: true
+            }
+          }
+        },
+        {
+          data: mockResponseStream,
+          status: 200
+        },
+        endpoint
+      );
+      jest.spyOn(AzureOpenAiChatClient.prototype, '_streamResponseChunks');
```

Comment:
Relevant to my removing sample-code comment, this unit test is good enough. For user, we maybe need to add documentation, or link it to LangChain's documentation since we did the same for other LangChain default behavioursl.